service/src/main/resources/WEB-INF/templates/fee-decision/voucher-value-decision_sv.properties

```diff
@@ -30,8 +30,8 @@ decision.details.child=För ditt barn {0} {1} {2} har beviljats rätt till servi
 decision.details.incomeChanges=Förändringar i inkomsterna<br /> Om det sker väsentliga förändringar i familjens inkomster (+/-10 %), ska en ny inkomstutredning göras omedelbart. Avgiften justeras från ingången av följande månad.
 
-decision.legalInstructions=<p>Tillämpade bestämmelser<br/> Lagen om klientavgifter inom småbarnspedagogiken (1503/2016)<br/> 7–9 § i lagen om servicesedlar inom social- och hälsovården (569/2009)</p> <p>Stadsstyrelsens beslut 9.11.2020 § 373 gällande värdet, pristaket och koefficienterna för servicesedeln inom småbarnspedagogiken<br /> Esbo stads servicesedelregelbok för småbarnspedagogik</p> <p>Behörighet<br/> Beslutanderätt i enlighet med lagstiftningen som gäller småbarnspedagogik och utbildning för tjänstemän inom Esbo stads resultatenhet Svenska bildningstjänster och bildningssektorns stab del B 2 § punkt 2</p>
-decision.relief.legalInstructions=<p>Tillämpade bestämmelser<br/> Lagen om klientavgifter inom småbarnspedagogiken (1503/2016)<br/> 8 § i lagen om servicesedlar inom social- och hälsovården (569/2009)</p> <p>Stadsstyrelsens beslut 6.3.2017 § 70 gällande principer för sänkning eller befrielse av småbarnpedagogikens klientavgift<br /> Esbo stads servicesedelregelbok för småbarnspedagogik</p> <p>Behörighet<br/> Beslutanderätt i enlighet med lagstiftningen som gäller småbarnspedagogik och utbildning för tjänstemän inom Esbo stads resultatenhet Svenska bildningstjänster och bildningssektorns stab del B 2 § punkt 2</p>
+decision.legalInstructions=<p>Tillämpade bestämmelser<br/> Lagen om klientavgifter inom småbarnspedagogiken (1503/2016)<br/> 7–9 § i lagen om servicesedlar inom social- och hälsovården (569/2009)</p> <p>Stadsstyrelsens beslut 9.11.2020 § 373 gällande värdet, pristaket och koefficienterna för servicesedeln inom småbarnspedagogiken<br /> Esbo stads servicesedelregelbok för småbarnspedagogik</p> <p>Behörighet<br/> Beslutanderätt i enlighet med lagstiftningen som gäller småbarnspedagogik och utbildning för tjänstemän inom Esbo stads resultatenhet svenska bildningstjänster och bildningssektorns sektor för fostran och lärande stab del B 2 § punkt 2</p>
+decision.relief.legalInstructions=<p>Tillämpade bestämmelser<br/> Lagen om klientavgifter inom småbarnspedagogiken (1503/2016)<br/> 8 § i lagen om servicesedlar inom social- och hälsovården (569/2009)</p> <p>Stadsstyrelsens beslut 6.3.2017 § 70 gällande principer för sänkning eller befrielse av småbarnpedagogikens klientavgift<br /> Esbo stads servicesedelregelbok för småbarnspedagogik</p> <p>Behörighet<br/> Beslutanderätt i enlighet med lagstiftningen som gäller småbarnspedagogik och utbildning för tjänstemän inom Esbo stads resultatenhet svenska bildningstjänster och bildningssektorns sektor för fostran och lärande stab del B 2 § punkt 2</p>
```

Comment:
Are you sure "Svenska bildningstjänster" isn't a proper name?

Reply:
It seems to vary, but most uses have a capital `S`. I changed this back now.